msmbuilder/tests/test_strucrmsdfeaturizer.py

```diff
@@ -38,10 +48,12 @@ def test_two_refs():
 
     dataset = fetch_alanine_dipeptide()
     trajectories = dataset["trajectories"]
-    featurizer = StrucRMSDFeaturizer(trajectories[0], trajectories[0][0:2],
+    featurizer = StrucRMSDFeaturizer(trajectories[0][0:2],
                                      range(trajectories[0].n_atoms))
     data = featurizer.transform(trajectories[0])
 
     # TODO: Figure out why arrays are 3D
```

Comment:
(I don't really know what it means). If it's no longer relevant you can get rid of it